api/application.go

```diff
@@ -1125,6 +1125,13 @@ func (r *Application) With(m *model.Application, tags []model.ApplicationTag) {
 		ref.With(a.ID, "")
 		r.Assessments = append(r.Assessments, ref)
 	}
+
+	if len(m.Analyses) > 0 {
+		sort.Slice(m.Analyses, func(i, j int) bool {
+			return m.Analyses[i].CreateTime.Before(m.Analyses[j].CreateTime)
+		})
+		r.Effort = m.Analyses[len(m.Analyses)-1].Effort
+	}
```

Comment:
Consider sorting by ID to be consistent with other queries (including the First(), Last() method on db).